frontend/templates/product/hooks/useProductReviews.ts

```diff
@@ -12,11 +12,7 @@ export function useProductReviews(product: Product) {
    const apiClient = useIFixitApiClient();
    const query = useQuery(
       productReviewsKeys.reviews(product.iFixitProductId),
-      () => fetchProductReviews(apiClient, product.iFixitProductId),
-      {
-         initialData: product.reviewsData,
-         staleTime: Infinity, // Right now we're not interested in refreshing data client side
```

Comment:
I think we still want this `staleTime` setting. Otherwise, the reviews may be refetched too often:
![image](https://user-images.githubusercontent.com/26855/200442550-a2c8f9ed-e0ce-4f59-b842-d013d8088e8f.png)

Reply:
im not sure any of those cases are really that bad in this instance, but I can add it back in.